.changesets/remove-ruby-2-version-checks.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "change"
+---
+
+The Appsignal gem requires Ruby 3 or higher to run. We should be able to remove any Ruby version checks that query Ruby 2.7 or lower.
```

Comment:
```suggestion
The AppSignal gem requires Ruby 3 or higher to run. Remove unnecessary Ruby version checks that query Ruby 2.7 or lower.
```

Small suggestion for improving the changeset text. I'll update it after the merge.